controllers/backstage_controller_test.go

```diff
@@ -56,10 +56,12 @@ var _ = Describe("Backstage controller", func() {
 		Expect(err).To(Not(HaveOccurred()))
 
 		backstageReconciler = &BackstageReconciler{
-			Client:      k8sClient,
-			Scheme:      k8sClient.Scheme(),
-			Namespace:   ns,
-			OwnsRuntime: true,
+			Client:         k8sClient,
+			Scheme:         k8sClient.Scheme(),
+			Namespace:      ns,
+			OwnsRuntime:    true,
+			PsqlImage:      "quay.io/fedora/postgresql-15:latest",
+			BackstageImage: "quay.io/janus-idp/backstage-showcase:next",
```

Comment:
if we use the restricted access (private repo) `quay.io/rhdh/rhdh=hub-rhel9:next` image here, we might be more easily able to test airgap, since OOTB a cluster won't be able to pull that image, but once copied inside the cluster's internal container registry, the image WILL be resolvable without the need for a quay pull secret.